system-addon/content-src/components/TopSites/TopSite.jsx

```diff
@@ -270,15 +270,19 @@ export class TopSitePlaceholder extends React.PureComponent {
 }
 
 export class _TopSiteList extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = this.DEFAULT_STATE = {
+  static get DEFAULT_STATE() {
+    return {
+      activeIndex: null,
```

Comment:
This is a relatively large behavior change compared to the rest of the changes in this PR. This ends up returning a new object every time it's "get"ed. We also haven't used static methods elsewhere…

Reply:
There's only static methods for now.. unless we babel it ;)